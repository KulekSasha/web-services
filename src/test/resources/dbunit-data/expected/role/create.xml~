<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <PERSON_ROLE ROLE_ID="1" NAME="Admin"/>
  <PERSON_ROLE ROLE_ID="2" NAME="User"/>
  <PERSON_ROLE ROLE_ID="3" NAME="NewRole"/>
</dataset>
