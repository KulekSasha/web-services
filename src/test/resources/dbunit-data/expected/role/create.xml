<?xml version='1.0' encoding='UTF-8'?>
<dataset>
    <PERSON_ROLE ROLE_ID="1" NAME="Admin"/>
    <PERSON_ROLE ROLE_ID="2" NAME="User"/>
    <PERSON_ROLE ROLE_ID="10" NAME="RoleForRemove"/>
    <PERSON_ROLE ROLE_ID="11" NAME="NewRole"/>
</dataset>
